buildkite/bazelci.py

```diff
@@ -3946,6 +3982,13 @@ def main(argv=None):
 
             platform = get_platform_for_task(args.task, task_config)
 
+            # TODO(https://github.com/bazelbuild/continuous-integration/issues/1800): Remove once we've processed the backlog of high-priority tasks.
+            if "mac" in platform and not is_high_priority_backlog_task():
+                raise BuildkiteException(
```

Comment:
Should we ignore the job instead of failing the build?

Reply:
Done